cmd/wksctl/profile/enable/enable.go

```diff
@@ -78,6 +82,23 @@ func profileEnableRun(params profileEnableFlags) error {
 	}
 	log.Info("Added the profile to the local repository.")
 
+	ignoreFilePath := path.Join(clonePath, constants.WKSctlIgnoreFilename)
+	if _, err := os.Stat(ignoreFilePath); err == nil {
+		log.Info("Ignoring files declared in .wksctlignore...")
+		file, err := os.Open(ignoreFilePath)
+		if err != nil {
+			return err
+		}
+		pathsToIgnores, err := processIgnoreFile(clonePath, file)
+		if err != nil {
+			return err
+		}
+		if err := removePathsFromGit(pathsToIgnores...); err != nil {
+			return err
+		}
+		log.Info("Ignored files successfully.")
+	}
```

Comment:
This whole block should go to a function on its own.

Reply:
refactored